### PR TITLE
London Class 7 - Chris Owen - Git - 1

### PR DESCRIPTION
**Pull request by ChrisOwen101**
# Your Details

- Your Name: Chris Owen
- Your City: London
- Your Slack Name: ChrisOwen

# Homework Details

- Module: Git
- Week: 1

### CODE REVIEW

**Review by Pat-On — COMMENTED** [review]
with flying colors :>

**Review by liewteh — COMMENTED** [review]
#1 corrected 'my kid's homework'

well done

File: homework.md [hunk 1]
```diff
@@ -92,13 +93,13 @@ A place for us to store and share our code.
...
+Save a change that you have made to your local repository using git
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+VS Code
 
 ## Question 17
```
Comment (bigbaby01) [inline homework.md:105]: well done this is brilliant

Comment (Pat-On) [reply homework.md:105]: definitely there should be NOTE!

**Comment by bita2019** [comment]
I didn't find any wrong things .

File: homework.md (hunk 1 above)
Comment (bita2019) [reply homework.md:105]: Keep going.

**Pull request closed without merging**